### DISCUSSION TITLE
Survival only break blocks in claims

### DISCUSSION THREAD

**Discussion by XxRosiexX** [GriefPrevention/GriefPrevention#734] [Q&A] [answered]
I'm trying to make it so you can only build/break blocks in the area you've claimed. Setting it to creative sorta works but doesn't drop items. is there any way around this? also the unclaim restore is very broken lol

**Comment by bigpresh** [comment]
SurvivalRequiringClaims should do what you want.

**Comment by XxRosiexX** [comment]
would i put that as mode?

**Comment by bigpresh** [comment, marked as answer]
Yes, it's one of the possible values for claims.mode in the config, documented at:
 https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on

So instead of e.g. `world: Survival` under `Claims -> Mode` in your config, you'd have `world: SurvivalRequiringClaims`

e.g.

```
GriefPrevention:
  Claims:
    Mode:
      world: SurvivalRequiringClaims
```

**Comment by XxRosiexX** [comment]
Thank you <3
